Gulpfile.coffee

```diff
@@ -190,5 +212,34 @@ gulp.task 'serve', ['watch'], ->
                 res.setHeader('Content-Type','image/svg+xml');
             next()
     ]
-
   connect.server(config)
+
+gulp.task '_tdd', (done) ->
+  buildAndTest(true, done)
+  return # Since this is async
+
+
+# -----------------------------------------------------------------------
+#  Public Tasks
```

Comment:
It was convenient to grep and replace all the names of private tasks. thoughts on naming this other file (`gulp-private-tasks.coffee`?

Or, a `gulp` directory in the root and files named `javascript-tasks.coffee`, `style-tasks.coffee`, `prod-tasks.coffee`? This might obscure the fact that `prod-tasks.coffee` depends on tasks defined in other files.